desktop/core/src/desktop/lib/botserver/urls.py

```diff
@@ -19,5 +19,5 @@
 from desktop.lib.botserver import views
 
 urlpatterns = [
-  url(r'^events/', views.home, name='home')
+  url(r'^events/', views.slack_events, name='slack_events')
```

Comment:
Should do only when enabled

Reply:
This file is getting added in urlpatterns when enabled

Reply:
Indeed!